website/docs/using-unleash/deploy/license-keys.mdx

```diff
@@ -1,119 +1,83 @@
 
----
 
-## Installing a new or replacing an existing license
+#### Seat count
+If you reach the maximum number of seats included in your license, you cannot add new users until existing users are removed to stay within your limit.
 
-Once you have a new or changed key, two methods are available to install a license on an enterprise self-hosted instance:
 
-1. Through the [**admin UI**](#admin-ui), logged in as an admin privileged user
-2. Specifying a [**environment variable**](#environment-variable) upon container deployment
+The Unleash Admin UI displays a banner if you do not comply with your license. In addition, you receive a warning in advance if less than 10% of your available seats remain, your license is 30 days or less from expiration, or if you are running on a trial license.
 
-The process is the same whether you are licensing your self-hosted instance for the first time.
-
-:::note
-After updating a license key, a manual browser refresh may be required for any displayed licensing related banners to be removed from your session.
-:::
+## Get a new license
 
----
+If you are a new customer, your account representative will provide the required license key as part of your self-hosted instance onboarding.
 
-### Admin UI
+If you are an existing customer and are making changes to your agreement (changing seat count or the contract expiration), contact your account representative to obtain the required license key.
 
-:::info
-This area of the UI can also be used to verify the currently installed key details
-:::
+Alternatively, you can reach out to sales@getunleash.io.
 
-1. Log in to the Unleash UI as an admin privileged user
-2. Navigate to the _admin menu (gear icon) -> License_
-3. If a key is already installed, the following details will be shown:
+## Check your current license
 
+To check your current license in the Unleash Admin UI, go to **Admin** > **License**. If you have an active license, you'll see the following information:
     - Customer name
     - Plan
     - Seat count
     - License expiration date
 
-4. Paste the new license key and click _Update license key_
+## Update your license
 
-![License overview screen](/img/license-overview.png)
+### Using the Admin UI
 
----
+In the Unleash Admin UI, do the following:
+1. Go to **Admin** > **License**.
+2. Paste the new license key and click **Update license key**.
+
+After updating a license key, you may need to refresh your browser for the license warning banners to be removed.
 
-### Environment Variable
+### Using an environment variable
 
-License key installation or update through an environment variable on container deployment is also possible, using the format `UNLEASH_LICENSE`.
+You can also install or update your license key on container deployment, using the environment variable `UNLEASH_LICENSE`.
```

Comment:
```suggestion
You can also install or update your license key on container deployment, using the environment variable `UNLEASH_LICENSE`.

If you try to update your license with an invalid one, or one that was issued before your existing license, the existing license will prevail.
```
Not sure if this is well written, but I can foresee questions about "what happens if" and this could clarify some of them.